config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  
-  devise_for :users
+  mount_devise_token_auth_for 'User', at: 'auth'
+
+  #devise_for :users
```

Comment:
Can we have both `devise_for` and `mount_devise_token_auth_for` in the same model? 
 - If we can, I wouldn't leave the `devise_for` commented. 
 - If we can't and `mount_devise_token_auth_for` is enough for both devise and devise_token_auth, I'd remove the line altogether